app.js

```diff
@@ -1,7 +1,37 @@
 const express = require("express")
 const { getApi } = require("./controllers/app.controller")
+const { getTopics } = require("./controllers/topics.controller")
 const app = express()
 
+app.use((req, res, next) => 
+    next()
+)
+
 app.get("/api", getApi)
 
+app.get("/api/topics", getTopics)
+
+
+
+app.use('/*', (req, res) => {
+    res.status(404).send({ msg: 'Route not found' })
+})
+
+app.use((err, req, res, next) => {
+    if (err.status){
+      res.status(err.status).send({ msg: err.msg })
+    } else next(err)
+  })
+  
+  app.use((err, req, res, next) => {
+    //console.log(err.code)
+    if (err.code){
+      res.status(400).send({ msg: 'Bad request' })
```

Comment:
If it has a key of `code`, it's likely coming straight from PSQL... Things to consider here:
A) Is it guaranteed to be coming from PSQL? Even if we add things to this server in the future?
B) Does a PSQL rejection always mean it's a bad request? What about it an internal server error happened?